code/modules/clothing/gloves/colored_gloves.dm

```diff
@@ -10,6 +10,7 @@
 	permeability_coefficient = 0.05
 	item_color="yellow"
 	resistance_flags = NONE
+	dyeable = TRUE
```

Comment:
also isn't technically needed, but its fine for specifics I guess

Reply:
correct, I'm just being safe